src/oas.md

```diff
@@ -2079,7 +2079,7 @@ headers:
       type: integer
 ```
 
-Response with no return value:
+A response with no return value can be specified by omitting the `content` field, although this technically does not enforce a lack of content, it merely indicates that none is documented:
```

Comment:
"no return value" is not very clear (some might say a response's "return value" is its HTTP response code) -- we should say "with no message body", or perhaps "zero length response body".